modules/rosa-deleting-cluster.adoc

```diff
@@ -143,3 +143,9 @@ $ rosa delete operator-roles -c <cluster_id> --mode auto <1>
 ----
 <1> Replace `<cluster_id>` with the ID of the cluster.
 endif::sts[]
+
+.Troubleshooting
+* If the cluster cannot be deleted because of missing IAM roles, see xref:../sd_support/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-cluster-deletion_rosa-troubleshooting-cluster-deployments[Repairing a cluster that cannot be deleted].
+* If the cluster cannot be deleted for other reasons:
+** Check that there are no Add-ons for your cluster pending in the link:https://console.redhat.com/openshift[Hybrid Cloud Console].
+** Check that all AWS resources and dependencies have been deleted in the Amazon Web Console.
```

Comment:
In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.html#rosa-deleting-cluster_rosa-sts-deleting-cluster